### ISSUE TITLE
Don't block `max-witdh` any more, as translate.w.org is becoming full screen.

### ISSUE THREAD

**Issue by vlad-timotei** [Mte90/GlotDict#403]
There are some experiments to convert [translate.wordpress.org](http://translate.wordpress.org/) to full width

Translation table – [current](https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/gl/default/?filters%5Btranslated%5D=yes&filters%5Bstatus%5D=current) / [full-width](https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/gl/default/?filters%5Btranslated%5D=yes&filters%5Bstatus%5D=current&full-width=true)
Translating a string – [current](https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/gl/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=14886966&filters%5Btranslation_id%5D=99633466) / [full-width](https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/gl/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=14886966&filters%5Btranslation_id%5D=99633466&full-width=true) (the design needs to be adjusted later for the right sidebar)

but currently GlotDict sets a fixed max-width to 85% so users using GlotDict will not see any changes. 

See here source: https://github.com/Mte90/GlotDict/blob/master/css/style.css#L718

There is no perfect percent for this value, but since translate.w.org increased this value I think we should restore it to the default value. Unless we have the resources and time to create a custom setting for this one and set it dynamically per user. 

What do you think?

**Comment by vlad-timotei** [comment]
Tickets to track
- https://meta.trac.wordpress.org/ticket/6594
- https://github.com/WordPress/wordpress.org/pull/111

**Comment by Mte90** [comment]
I think to remove it as it is more simple.

**Comment by psmits1567** [comment]
Yes I would suggest to remove it as well, which I can do and test

**Comment by psmits1567** [comment]
I have removed it, and it does not make any difference. So it is safe to remove it

**Comment by psmits1567** [comment]
Available in branch psmits1567-patch-3
